packages/pwa/app/hooks/use-navigation.js

```diff
@@ -26,7 +27,22 @@ const useNavigation = () => {
          * @param  {...any} args - additional args passed to `.push` or `.replace`
          */
         (path, action = 'push', ...args) => {
-            const updatedHref = buildPathWithUrlConfig(path, {locale})
+            // sanitize the locale/site from the current relative url
+            let [pathname, search] = path.split('?')
+            pathname = pathname.replace(`/${site.alias || site.id}`, '').replace(`/${locale}`, '')
+            search = search
+                ?.replace(`locale=${locale}`, '')
+                ?.replace(`site=${site.alias || site.id}`, '')
```

Comment:
With Ben's recent update to default.js config, there is no longer `site.alias` 🤔 Maybe bring it back to the config?

Reply:
Oh I think I see now: `site.alias` would be available because the alias is generated from `getSites()`?

Reply:
Yes, since we the alias is not inside the site from the config as we want to separate the data from demandware and pwa-kit. We use `siteAliases` to map the site id. In the code, the `getSites` will merge the alias to each site so it can easily be used across the app